integration_tests/fixtures/bastion/main.tf

```diff
@@ -76,13 +76,7 @@ resource "google_compute_firewall" "main" {
 
   direction     = "INGRESS"
   priority      = "100"
-
-  # It is okay to leave this open to everyone as SSH key is also required to
-  # SSH into the server VM and SSH key is generated dynamically. Also, the host
-  # is ephemeral and the environment itself is deconstructed soon after tests
-  # are executed. Narrowing it down to few IP addresses makes it difficult for
-  # the developers to test on their local dev environment.
-  source_ranges = ["0.0.0.0/0"]
+  source_ranges = ["${bastion_firewall_netblocks}"]
```

Comment:
The bastion_firewall_netblocks variable should be prepended by `var` (`"${var.bastion_firewall_netblocks}"`) as it tells Terraform what that value is (i.e. it's a variable, so should be found in `variables.tf`)

Since this is already a list, should be ok to just have it on its own, i.e. `source_ranges = "${var.bastion_firewall_netblocks}"`

Reply:
Fixed. Thanks